website/client/src/components/sidebarButton.vue

```diff
@@ -28,13 +28,18 @@
     background: transparent;
     cursor: pointer;
     &:focus {
+      // Fix keyboard inaccessible #12656
```

Comment:
Should this be an issue link in the PR instead of a code comment?

Reply:
Either I copied from a different place - or the merge of develop added this - I don't think I wrote that on purpose :D 

I can replace it with a github link - but I also need to find the source of that as well

Reply:
oh it was at first a conflict and then I added the comment to know why outline was needed - now I remember hah